src/annotator/integrations/vitalsource.js

```diff
@@ -214,12 +214,10 @@ export class VitalSourceContentIntegration {
     // from showing its native selection menu, which obscures the client's
     // annotation toolbar.
     //
-    // VitalSource only checks the selection on the `mouseup` and `mouseout` events,
-    // but we also need to stop `mousedown` to prevent the client's `SelectionObserver`
-    // from thinking that the mouse is held down when a selection change occurs.
-    // This has the unwanted side effect of allowing the adder to appear while
-    // dragging the mouse.
-    const stopEvents = ['mousedown', 'mouseup', 'mouseout'];
+    // To avoid interfering with the client's own selection handling, this
+    // event blocking must happen at the same level or higher in the DOM tree
+    // than where SelectionObserver listens.
+    const stopEvents = ['mouseup', 'mouseout'];
```

Comment:
Does this mean that `mousedown` events were previously not-propagated and now are set free? Or was it  that before, because the listener was "too high up" on the `document` instead of `document.body`, `mousedown` events weren't getting captured the way we thought they were in the first place? In case that doesn't make any sense: Why was `mousedown` removed from this list :) ?

Reply:
Previously SelectionObserver listened at a higher level than this code blocks events. This meant that it was necessary for the blocking to be "symmetric" by blocking both `mouseup` and `mousedown` to avoid causing SelectionObserver to get into the wrong state. Now this has changed so SelectionObserver listens at a lower level. Hence only events that affect VS need to be blocked at this level - which are `mouseup` and `mouseout`.